packages/orbit-design-tokens/src/__tests__/deprecatedTokens.test.ts

```diff
@@ -0,0 +1,592 @@
+    backgroundTagSelected: theme.palette.ink.dark,
+    backgroundTagHover: theme.palette.cloud.lightHover,
+    backgroundTagSelectedHover: theme.palette.ink.darkHover,
+    backgroundTagActive: theme.palette.cloud.lightActive,
+    backgroundTagSelectedActive: theme.palette.ink.darkActive,
+    backgroundTooltip: theme.palette.white.normal,
+    backgroundTooltipLargeMobile: theme.palette.blue.dark,
+    // category:Font size
+    // description:Use these sizing tokens for font sizes.
+    fontSizeHeadingDisplay: "40px",
+    fontSizeHeadingDisplaySubtitle: "22px",
+    fontSizeHeadingTitle1: "28px",
+    fontSizeHeadingTitle2: "22px",
+    fontSizeHeadingTitle3: "18px",
+    fontSizeHeadingTitle4: theme.base.fontSizeLg,
+    fontSizeHeadingTitle5: theme.base.fontSizeMd,
+    fontSizeHeadingTitle6: theme.base.fontSizeSm,
+    fontSizeTextNormal: theme.base.fontSizeMd,
+    fontSizeTextLarge: theme.base.fontSizeLg,
+    fontSizeTextSmall: theme.base.fontSizeSm,
+    fontSizeMessage: theme.base.fontSizeMd,
+    fontSizeButtonLarge: theme.base.fontSizeLg,
+    fontSizeButtonNormal: theme.base.fontSizeMd,
+    fontSizeButtonSmall: theme.base.fontSizeSm,
+    fontSizeInputNormal: theme.base.fontSizeMd,
+    fontSizeInputSmall: theme.base.fontSizeMd,
+    fontSizeFormLabel: theme.base.fontSizeMd,
+    fontSizeFormFeedback: theme.base.fontSizeSm,
+    // category:Border radius
+    // description:Use radius tokens to change the border-radius size (rounded corners).
+    /** use only for square */
+    borderRadiusCircle: "50%",
+    /** use for all basic elements like buttons, inputs, content containers */
+    borderRadiusNormal: theme.base.borderRadius,
+    /** use for big elements like modal and large buttons */
+    borderRadiusLarge: "6px",
+    /** use for smaller elements like tags */
+    borderRadiusSmall: "2px",
+    borderRadiusBadge: "12px",
+    // category:Z Index
+    // description:Use layers to show depth of page.
+    zIndexDefault: "1",
+    zIndexSticky: "100",
+    zIndexDrawer: "815",
+    zIndexModalOverlay: "800",
+    zIndexModal: "825",
+    zIndexOnTheTop: "900",
+    // category:Size (width, height)
+    // description:
+    widthIconSmall: theme.base.iconSizeSm,
+    heightIconSmall: theme.base.iconSizeSm,
+    widthIconMedium: theme.base.iconSizeMd,
+    heightIconMedium: theme.base.iconSizeMd,
+    widthIconLarge: theme.base.iconSizeLg,
+    heightIconLarge: theme.base.iconSizeLg,
+    heightInputNormal: theme.base.sizeXl,
+    heightInputLarge: theme.base.size2xl,
+    heightInputSmall: theme.base.sizeLg,
+    heightButtonNormal: theme.base.sizeXl,
+    heightButtonLarge: theme.base.size2xl,
+    heightButtonSmall: theme.base.sizeLg,
+    heightCarrierLogo: theme.base.sizeLg,
+    widthCarrierLogo: theme.base.sizeLg,
+    heightRadioButton: "20px",
+    widthRadioButton: "20px",
+    heightCheckbox: "20px",
+    widthCheckbox: "20px",
+    heightCountryFlag: "auto",
+    widthCountryFlag: theme.base.sizeMd,
```

Comment:
This is refering to sizeMd, which is 24px, but on the generated `createTokens` on the test snapshot and on the output folder it has a value of 20px 😕 can you further investigate why? This is deprecated and we won't need this token anymore anyway, but it would be interesting to understand why this is happening, because it can be happening somewhere else 💭